core-lib/TestSuite/LiteralObjectTests.som

```diff
@@ -0,0 +1,102 @@
+(* Copyright (c) 2001-2015 see AUTHORS file
```

Comment:
also, you might want to add yourself to the AUTHORS file while you're at it.
And, the CHANGELOG.md deserves an entry, too :)